crates/sui-adapter/src/programmable_transactions/context.rs

```diff
@@ -365,6 +365,11 @@ where
         Ok(id)
     }
 
+    /// Add a package to the context.
+    pub fn add_package(&mut self, package_object: Object) {
+        self.new_packages.push(package_object)
+    }
+
```

Comment:
If you are going to do this, could you refactor `new_package` out? I'd rather not have two functions that push to `new_packages`.